<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_setproxy</class>
 <widget class="QWidget" name="Form_setproxy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>326</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Set Proxy</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/filmonaizer.qrc">
    <normaloff>:/icons/globe.png</normaloff>:/icons/globe.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_proxy">
       <property name="text">
        <string>Enable proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Settings proxy</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,10,0">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>List proxy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="list_proxy"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>IP proxy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_ip_proxy"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port proxy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_port_proxy"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Username proxy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_user_proxy"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Password proxy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_pass_proxy"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="button_add_proxy">
          <property name="text">
           <string>Add proxy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_save_proxy">
          <property name="text">
           <string>Save proxy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_del_proxy">
          <property name="text">
           <string>Delete Proxy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_choose">
       <property name="text">
        <string>Choose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_proxy</tabstop>
  <tabstop>text_ip_proxy</tabstop>
  <tabstop>text_port_proxy</tabstop>
  <tabstop>text_user_proxy</tabstop>
  <tabstop>text_pass_proxy</tabstop>
  <tabstop>list_proxy</tabstop>
  <tabstop>button_add_proxy</tabstop>
  <tabstop>button_save_proxy</tabstop>
  <tabstop>button_del_proxy</tabstop>
  <tabstop>button_choose</tabstop>
  <tabstop>button_close</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/filmonaizer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox_proxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>Form_setproxy</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
